### ISSUE TITLE
Locations list not accessible via keyboard

### ISSUE THREAD

**Issue by BlueSlug** [inclusive-design/covid-data-monitor#16]
## Describe the bug
The "Locations" list items are not accessible via keyboard navigation. This could have significant usability implications for many users, including those who rely on assistive technologies such as switches or screen readers. This issue was noted on both desktop and mobile devices

### To reproduce
1. Go to the Map page: https://wecount.inclusivedesign.ca/map
2. Press the "Tab" key to navigate through the interactive elements of the site
3. Note that the items on the "Locations" list are not accessible. In Firefox, the panel can be scrolled up and down via the keyboard, though the list items themselves remain inaccessible.

### Expected behavior
The list items should be included in the tab order and should be operable via keyboard only

## Screenshots
<img width="357" alt="2021-06-12 wecount a11y map location no kb" src="https://user-images.githubusercontent.com/547933/121828208-ebe3e180-cc8c-11eb-80b3-ea2329f9d4ac.PNG">

### Desktop
 - OS: Windows 10 v20H2
 - Browser: Chrome 91, Firefox 89

### Smartphone / tablet 
 - Device: Pixel 5
 - OS: Android 11
 - Browser: Chrome 91

**Comment by cindyli** [comment]
The same issue also happens to the search result list, which shows up when a location is selected.
